### PR TITLE
Create a script to load videos from youtube API

### LINKED ISSUE #627: Create a script to load videos from youtube API
**Is your feature request related to a problem? Please describe.**
- The dummy video archive page needs to be replaced with data.

**Describe the solution you'd like**
- Write a script to load items from the url.
- Refer to how mentors profiles are loading in scholarX to get a basic idea.
- Use local copy of this https://sef-dataholder.herokuapp.com/past-one-lives

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #627 

## Goals

- Create a script to load videos from youtube API

## Approach

- Write a script to load items from the url.

### Preview Link

https://pr-628-sef-site.surge.sh/onelive/archive

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/onelive/archive/index.html [hunk 1]
```diff
@@ -154,6 +83,58 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
...
+            dataType: 'json',
+            success: function (data) {
+
+                //mustache render
+                let renderedData = Mustache.render(
+                    $('#template-youtube-vidoes').html(), {data : data});
+
+                    $('#youtube-video').html(renderedData);
+            }
+        });
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/index.html]: Remove extra spaces. Format it properly. 
Add meaningful ids. :)

File: src/main/webapp/onelive/archive/index.html [hunk 2]
```diff
@@ -42,87 +44,14 @@ <h1 class="mt-5 text-onelive" style="font-size: 3.3rem; line-height: 1.5;"><span
...
+        <div class="container" id="youtube-video">
+        <!--youtube video goes here-->
+        </div>
+    </section>
+
+    <section class="section bg-secondary">
+        <div class="container" id="youtube-first">
+            <!--youtube video goes here-->
         </div>
     </section>
```
Comment (Piumal1999) [inline src/main/webapp/onelive/archive/index.html]: Do we need this section?

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Nice!!
It is good if we can keep the CSS file in the main assets/css directory. 
And also format the code.

**Comment by jayasanka-sack** [comment]
> It is good if we can keep the CSS file in the main assets/css directory.

@Piumal1999 It has been changed with the new file structure. If a page using custom styles that are used by only that page, has to be in the same directory. 

But, @anjula-sack could you please explain what line 15  - 456 does on the style.css? Is there any specific reason?

File: src/main/webapp/onelive/archive/index.html [hunk 3]
```diff
@@ -154,6 +83,58 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
...
+        </div>
+        {{/data.items}}
+    </div>
+    {{#data}}
+    <div class="row justify-content-center mb-md-5">
+        <div class="col-9 text-center">
+            <a class="btn btn-onelive text-uppercase" onclick="setPageToken('{{nextPageToken}}')">show more</a>
+        </div>
+    </div>
+    {{/data}}
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/index.html]: Remove this section because the logic isn't implemented.

**Comment by Piumal1999** [comment]
>
> @Piumal1999 It has been changed with the new file structure. If a page using custom styles that are used by only that page, has to be in the same directory.
> 

Ah ok. :grinning:

File: src/main/webapp/onelive/archive/index.html (hunk 2 above)
Comment (anjula-sack) [reply src/main/webapp/onelive/archive/index.html]: no my bad

**Review by Piumal1999 — APPROVED** [review]

File: src/main/webapp/onelive/archive/index.html [hunk 4]
```diff
@@ -154,6 +77,54 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
...
+    function loadYoutubeVideos() {
+        $.ajax({
+            type: 'get',
+            url: 'https://sef-dataholder.herokuapp.com/youtube/onelive',
+            dataType: 'json',
+            success: function (data) {
+                //mustache render
+                let renderedData = Mustache.render(
+                    $('#template-youtube-vidoe').html(), {data: data});
+                $('#youtube-video').html(renderedData);
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/index.html]: videos

**Pull request merged**